gems/sorbet-runtime/lib/types/props/decorator.rb

```diff
@@ -247,18 +247,18 @@ def prop_validate_definition!(name, cls, rules, type)
     nil
   end
 
-  SAFE_NAME = T.let(/\A[A-Za-z_][A-Za-z0-9_-]*\z/.freeze, Regexp)
+  SAFE_NAME = T.let(/\A[A-Za-z_][A-Za-z0-9_-]*\z/.freeze, Regexp, checked: false)
```

Comment:
Just to clarify, we are `checked: false`'ing all of these so that we don't pay the slight cost of typechecking the top-level constants at code-loading time?  This seems...overkill to me, but I'm not exercised about it to block this PR.